api/controllers/reports.js

```diff
@@ -0,0 +1,232 @@
+'use strict';
+
+var _ = require('lodash');
+var async = require('async');
+var User = require('../../models/User');
+var Hours = require('../../models/Hours');
+var weeksSinceEpoch = require('../../lib/util').weeksSinceEpoch;
+
+/**
+ * Gets hours for an epoch window (optionally), and returns them in flat layout.
+ */
+
+var getHours = function (epochSubtractor, callback) {
```

Comment:
can do. Since your PR has it this way, I think we should each just do a static for what we need, then refactor them together later once they are both merged if needed. I calculate totals in the view layer, and yours is a component of the db pull.